{"uuid": "1e03fef9-10d7-4fbe-a41d-85b1d33d9424", "counter": 53, "stdout": "\u001b[0;31mfatal: [demo03]: FAILED! => {\"changed\": false, \"msg\": \"One or more supplied key could not be found in the database.\"}\u001b[0m\r\n\u001b[0;36m...ignoring\u001b[0m", "start_line": 57, "end_line": 59, "runner_ident": "2363d8ab-498d-450d-aa97-91ef4bad8344", "event": "runner_on_failed", "pid": 229719, "created": "2025-05-16T14:01:46.063310+00:00", "parent_uuid": "005056b7-f67d-ca61-7db1-000000000014", "event_data": {"playbook": "/tmp/tmp6sa_kl36.yml", "playbook_uuid": "e5fbbb2e-7663-4699-85dd-3157f1a5aa28", "play": "Instalaci\u00f3n y configuraci\u00f3n de Pure-FTPd para hosts", "play_uuid": "005056b7-f67d-ca61-7db1-000000000002", "play_pattern": "demo03", "task": "Verificar si el usuario FTP existe", "task_uuid": "005056b7-f67d-ca61-7db1-000000000014", "task_action": "getent", "resolved_action": "ansible.builtin.getent", "task_args": "", "task_path": "/tmp/tmp6sa_kl36.yml:149", "host": "demo03", "remote_addr": "demo03", "res": {"msg": "One or more supplied key could not be found in the database.", "invocation": {"module_args": {"database": "passwd", "key": "ftpuser", "fail_key": true, "service": null, "split": null}}, "_ansible_no_log": false, "changed": false}, "start": "2025-05-16T14:01:45.081050+00:00", "end": "2025-05-16T14:01:46.062935+00:00", "duration": 0.981885, "ignore_errors": true, "event_loop": null, "uuid": "1e03fef9-10d7-4fbe-a41d-85b1d33d9424"}}